monitoring/README.md

```diff
@@ -0,0 +1,98 @@
+# Monitoring
+
+This directory contains the monitoring dashboards for Accelerators and vLLM performance metrics. This includes:
+
+- [NVIDIA](grafana/nvidia/nvidia-vllm-dashboard.yaml) GPU metrics with vLLM Performance Metrics.
+- [inputs.env](inputs.env): holds the parameters that need to be updated before deploying the Grafana dashboard.
+
+The Grafana dashboard object is defined as a JSON string as this example:
+```yaml
+...
+spec:
+  instanceSelector:
+    matchLabels:
+      dashboards: grafana
+  folder: "vLLM / GPU Metrics"
+  json: |
+    {
+      "annotations": {
+        "list": [
+          {
+            "builtIn": 1,
+            "datasource": {
+              "type": "$(DATASOURCE)",
+              "uid": "grafana"
+            },
+            "enable": true,
+            "hide": true
+            "iconColor": "rgba(0, 211, 255, 1)",
+            "name": "Annotations & Alerts",
+            "type": "dashboard"
+          }
+        ]
+      }
+...
+#remaining of the JSON definition
+```
+
+With this approach, it makes difficult to use kustomize to correctly override the needed parameters so the Dashboard
+can correctly show the metrics deployed in your current namespace. There are two parameters that needs to be updated:
+
+- **NAMESPACE**: the targe namespace where the target model will be deployed.
+- **MODEL_NAME**: the same as defined in your InferenceService, note, it is a regex, so just the starting string of your model name would be enough.
+
+To correctly override it, we will use a helper called `envsubst`, it is available through the `gettext` GNU package.
+
+We will use the `envsubst` command to replace the variables in the JSON string with the actual values provided in the 
+[inputs.env]() file, example:
+
+```bash
+export $(cat config/grafana/nvidia/inputs.env | xargs)
```

Comment:
Can you add a quick sentence describing what this does? It reads a bit confusingly when you say we're going to use `envsubt` and then the next command doesn't do that.

Reply:
sure.